### ISSUE TITLE
Debugging elasticsearch-transport

### ISSUE THREAD

**Issue by ferbar** [uken/fluent-plugin-elasticsearch#341]
#### Problem

Elasticsearch-transport would log a lot if we gave it a logger object. This would make debugging much easier.

#### Expected Behavior or What you need to ask

Get error/debug messages from elasticsearch-transport.

#### Using Fluentd and ES plugin versions

* Fluentd v0.12
* ES plugin 1.9.5

#### Solutinon

To get the log messages from elasticsearch-transport yust jet logging object has to be passed.
See in file fluent/plugin/out_elasticsearch_dynamic.rb:46 adding:
`logger:log,`

would do the trick. Do we need special Options to enable/disable this?

**Comment by cosmo0920** [comment]
It sounds good to me!
But this option should be disabled by default.
Because POST or something always appears in log:

```log
2017-12-20 11:19:24 +0900 [info]: #0 POST http://localhost:9200/_bulk [status:200, request:0.014s, query:0.010s]
```

**Comment by cosmo0920** [comment]
I've released this enhancement as v1.13.1 and v2.4.1.
